src/commands/createManagedEnvironment/ManagedEnvironmentCreateStep.ts

```diff
@@ -43,9 +44,16 @@ export class ManagedEnvironmentCreateStep extends AzureWizardExecuteStep<IManage
             }
         );
 
+        context.activityResult = {
```

Comment:
We can also do this now. We can change this after the release.

```suggestion
        context.activityResult = nonNullProp(context.managedEnvironment, 'id')
```